vlm/src/test/scala/geotrellis/contrib/vlm/RasterSourceRDDSpec.scala

```diff
@@ -53,6 +54,8 @@ class RasterSourceRDDSpec extends FunSpec with TestEnvironment with BetterRaster
   val reprojectedSource = rasterSource.reprojectToGrid(targetCRS, layout)
 
   describe("reading in GeoTiffs as RDDs") {
+    GDALWarp.init(1<<8, 1<<2)
```

Comment:
A design question: should be always known ahead of time? For instance before the application start? Also what are the rules to select this numbers?

Reply:
Alot of what you see here is stub code.